src/odemis/driver/autoscript_client.py

```diff
@@ -0,0 +1,2161 @@
+
+    def get_field_of_view(self, channel: str) -> float:
+        """
+        :param channel: (str) Name of the channel to get the field of view for.
+        :return: (float) the field of view in meters.
+        """
+        with self._proxy_access:
+            self.server._pyroClaimOwnership()
+            return self.server.get_field_of_view(channel)
+
+    def field_of_view_info(self, channel: str) -> dict:
+        """Returns: (dict) the scanning size unit and range."""
+        with self._proxy_access:
+            self.server._pyroClaimOwnership()
+            return self.server.field_of_view_info(channel)
+
+    def set_high_voltage(self, voltage: float, channel: str) -> None:
+        """
+        Set the high voltage.
+        :param voltage: (float) the high voltage in volt.
+        :param channel: (str) Name of the channel to set the high voltage for.
+        """
+        with self._proxy_access:
+            self.server._pyroClaimOwnership()
+            self.server.set_high_voltage(voltage, channel)
+
+    def get_high_voltage(self, channel: str) -> float:
+        """:return (float) the high voltage in volt."""
+        with self._proxy_access:
+            self.server._pyroClaimOwnership()
+            return self.server.get_high_voltage(channel)
+
+    def high_voltage_info(self, channel: str) -> dict:
+        """:return (dict) the unit and range of the high voltage."""
+        with self._proxy_access:
+            self.server._pyroClaimOwnership()
+            return self.server.high_voltage_info(channel)
+
+    def set_beam_current(self, current: float, channel:str) -> None:
+        """
+        Set the beam current.
+        :param current: (float) the beam current in ampere.
+        :param channel: (str) Name of the channel to set the beam current for.
+        """
+        with self._proxy_access:
+            self.server._pyroClaimOwnership()
+            self.server.set_beam_current(current, channel)
+
+    def get_beam_current(self, channel: str) -> float:
+        """Returns: (float) the beam current in ampere."""
+        with self._proxy_access:
+            self.server._pyroClaimOwnership()
+            return self.server.get_beam_current(channel)
+
+    def beam_current_info(self, channel: str) -> dict:
+        """Returns: (dict) the unit and range of the beam current."""
+        with self._proxy_access:
+            self.server._pyroClaimOwnership()
+            return self.server.beam_current_info(channel)
+
+    def blank_beam(self, channel: str) -> None:
+        """Blank the selected beam."""
+        with self._proxy_access:
+            self.server._pyroClaimOwnership()
+            self.server.blank_beam(channel)
+
+    def unblank_beam(self, channel: str) -> None:
+        """Unblank the selected beam."""
+        with self._proxy_access:
+            self.server._pyroClaimOwnership()
+            self.server.unblank_beam(channel)
+
+    def beam_is_blanked(self, channel: str) -> bool:
```

Comment:
```suggestion
    def is_beam_blanked(self, channel: str) -> bool:
```
Just a comment

Reply:
it was named this way to match the same functions in the xt_client